vars/whitesourceExecuteScan.groovy

```diff
@@ -275,6 +280,18 @@ void call(Map parameters = [:]) {
             .addIfEmpty('whitesource/scanImageRegistryUrl', script.commonPipelineEnvironment.containerProperties?.registryUrl)
             .use()
 
+
+        if (config.useGoStep == true && config.scanType != "unified-agent") {
+            parameters = com.sap.piper.DownloadCacheUtils.injectDownloadCacheInParameters(script, parameters, BuildTool.MTA)
```

Comment:
You can import com.sap.piper.DownloadCacheUtils and remove the qualification. (This was necessary at the other place, but not here.)